### PR TITLE
chore(core): show "Deleting" when environment gets deleted

### PR DESCRIPTION
**Pull request by ranihorev**
Kapture 2021-08-17 at 12 29 02

### CODE REVIEW

File: packages/core/src/managed/environmentBaseElements/BaseEnvironment.less [hunk 1]
```diff
@@ -38,6 +39,18 @@
       font-weight: 700;
     }
 
+    .env-deleting {
+      position: absolute;
+      right: 8px;
+      font-size: 13px;
+      color: var(--color-white);
+      animation: deletingAnimation 2s infinite;
+      padding: var(~'--3xs-spacing') var(~'--xs-spacing');
```
Comment (luispollo) [inline packages/core/src/managed/environmentBaseElements/BaseEnvironment.less:48]: Just for my education, what's the `~` here?

File: packages/core/src/managed/overview/baseStyles.less [hunk 2]
```diff
@@ -118,3 +118,15 @@ button.md-btn-danger:hover {
     background-color: var(--color-status-success);
   }
 }
+
+@keyframes deletingAnimation {
```
Comment (luispollo) [inline packages/core/src/managed/overview/baseStyles.less:122]: Cool! Thanks for adding the animation like I suggested! ❤️

**Review by luispollo — APPROVED** [review]
Not sure why the build is failing, but looks good to me.

File: packages/core/src/managed/environmentBaseElements/BaseEnvironment.less (hunk 1 above)
Comment (ranihorev) [reply packages/core/src/managed/environmentBaseElements/BaseEnvironment.less:48]: LESS doesn't like the format `--3xs-spacing`, the `~` tells it to treat it as a string. It'll work without it but the UI will complain

**Pull request merged**